databricks_cli/stack/cli.py

```diff
@@ -46,7 +92,20 @@ def deploy(api_client, config_path, **kwargs):
     """
     click.echo('#' * 80)
     click.echo('Deploying stack at: {} with options: {}'.format(config_path, kwargs))
-    StackApi(api_client).deploy(config_path, **kwargs)
+    stack_config = _load_json(config_path)
+    status_path = _generate_stack_status_path(config_path)
+    stack_status = _load_json(status_path)
+    config_dir = os.path.dirname(os.path.abspath(config_path))
+    cli_dir = os.getcwd()
+    os.chdir(config_dir)  # Switch current working directory to where json config is stored
+    new_stack_status = StackApi(api_client).deploy(stack_config, stack_status, **kwargs)
+    click.echo('#' * 80)
+    os.chdir(cli_dir)
+    click.echo("Saving stack status to {}".format(status_path))
+    _save_json(status_path, new_stack_status)
+    click.echo('Warning- The stack status file is an automatically generated file that is'
```

Comment:
"WARNIN" usually mean there are some bad style, which does not break the program immediately. Here it would make more sense if you put those comments in the generated  "status" file. (Or just remove this warning in this PR).

Reply:
I would put it as a comment in the generated status file, but JSON doesn't allow for comments. Maybe I can put "Note:"?